test/git-lfs-test-server-api/main.go

```diff
@@ -147,17 +149,16 @@ func buildTestData() (oidsExist, oidsMissing []TestObject, err error) {
 	defer repo.Cleanup()
 	// just one commit
 	commit := test.CommitInput{CommitterName: "A N Other", CommitterEmail: "noone@somewhere.com"}
-	var totalSize int64
 	for i := 0; i < oidCount; i++ {
 		filename := fmt.Sprintf("file%d.dat", i)
 		sz := int64(rand.Intn(200)) + 50
 		commit.Files = append(commit.Files, &test.FileInput{Filename: filename, Size: sz})
-		totalSize += sz
+		meter.AddEstimate(sz)
```

Comment:
What do you think about leaving the `totalSize` variable and adding to that inside the loop, then calling the `AddEstimate` function once? If `oidCount` is large, I figure that the overhead of calling `atomic.Add<T>` functions will add up.